src/Donations/PaymentMethods/PaymentMethodTabs.tsx

```diff
@@ -79,7 +79,7 @@ export default function PaymentMethodTabs({
       case "trees":
         paymentLabel = t("treesInCountry", {
           treeCount: quantity,
-          country: t(`country:${projectDetails.country.toLowerCase()}`),
+          // country: t(`country:${projectDetails.country.toLowerCase()}`),
```

Comment:
from https://app.plant-for-the-planet.org/app/donations/don_xyz

project.country